README.rst

```diff
@@ -30,19 +30,29 @@ The image accepts a number of parameters in the form of environment
 variables (i.e. passed using the ``-e`` docker flag) that will
 configure the behavior of the container.
 
+MDB_FARM_DIR
+   The location of the database farm. Defaults to /var/monetdb5/dbfarm.
+   If unset, no farm will be created and no MonetDB daemon will be started.
+
 MDB_DAEMON_PASS
-   This is the passphrase used to contact the MonetDB daemon remotely,
-   i.e. from outside the container. For example::
+   This is the passphrase used to manage the farm from outside the container.
+   For example::
 
     monetdb -h localhost -P <passphrase> create SF-1
 
-   The default value is ``monetdb`` but we *strongly* advise you to set a
-   different value explicitly.
+   See also MDB_DAEMON_PASS_FILE, which may be more secure.
+
+   If neither MDB_DAEMON_PASS nor MDB_DAEMON_PASS_FILE are set,
+   the farm cannot be managed from outside the container
+
+MDB_DAEMON_PASS_FILE
+   Variant of MDB_DAEMON_PASS. Path to a file inside the container that contains
+   the daemon password.
```

Comment:
I think we should describe here the expected contents of the file, and maybe how to put the file in the container?